modules/ROOT/pages/index.adoc

```diff
@@ -3,25 +3,55 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: mule, getting started, transform, message, payload
+:page-aliases: mule-concepts.adoc
 
-Mule runtime engine (Mule) is a lightweight integration engine that runs Mule apps.
-Users can build Mule apps to connect systems, services, APIs, and devices using MuleSoft's
-API-led connectivity instead of point-to-point integrations.  Mule apps provide functionality
-for message routing, data mapping, orchestration, reliability, security, and scalability.
+Mule runtime engine (Mule) is a lightweight integration engine that runs Mule
+applications and supports domains and policies. Mule applications, domains,
+and policies share an XML DSL (domain-specific language).
 
-With Mule you can:
+== Mule Applications
 
-* Connect data to apps in Enterprise Service Bus (ESB) patterns.
-* Build APIs.
-* Access, query, and transform data through the DataWeave language.
-* Configure high-availability, clustering, and performance management at scale.
-* Deploy an integration worker, ESB or API gateway, on-premises or in the cloud.
-* Automatically manage thread pools with the self-tuning, reactive Mule runtime engine.
-* Use error constructs and try scopes for rapid debugging.
-* Isolate the classloader to protect Mule apps from changes to the runtime or connectors.
+Mule applications connect systems, services, APIs, and devices using API-led
+connectivity instead of point-to-point integrations. Mule applications provide
+functionality for message routing, data mapping, orchestration, reliability,
+security, and scalability.
+
+Anypoint Studio and Flow Designer support Mule application development.
+
+== Domains
+
+Domains enable you to share global configurations that Mule applications need to
```

Comment:
This tells what a domain does but do we want to say first what a domain is?

Reply:
Does this mean the same thing? It makes the sentence a bit more concise:
```suggestion
Domains enable you to share global configurations that Mule applications
```
"...reuse, such as....", next line